webplugin/js/app/mck-app.js

```diff
@@ -310,6 +310,12 @@ function ApplozicSidebox() {
             var widgetSettings = data.chatWidget;
             var disableChatWidget = options.disableChatWidget != null ? options.disableChatWidget : widgetSettings.disableChatWidget; // Give priority to appOptions over API data.
             
+            // exclude kommunicate.io from restricted domains for
+            // the chatbot preview feature
+            var kommunicateDomains = [];
```

Comment:
Can we directly write ["kommunicate.io"]

Reply:
@shubhamtibra Add this as constant on km-utils files.
ALLOWED_DOMAINS = [ "kommunicate.io"]
In future, we can add other domains if needed in this array only.

Reply:
will do